CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Changelog
 
+## [1.56](https://github.com/brave/brave-ios/releases/tag/v1.56)
+
+### Web3
+
+- Implemented “Select a token to send” view which allows selecting an account and an asset at the same time. ([#7523](https://github.com/brave/brave-ios/issues/7523))
+- Removed network name from the “Public Encryption Key Request” panel. ([#7495](https://github.com/brave/brave-ios/issues/7495))
+- Improved NFT importing in Brave Wallet by introducing a new button to add custom assets quicker in NFT tab view and also simplifying the “Add Custom Asset” form by moving moving “Token address” field to the top. ([#7556](https://github.com/brave/brave-ios/issues/7556))
+- Updated “Select a Token to Send” screen to sort test network tokens to be displayed at the end. ([#7640](https://github.com/brave/brave-ios/issues/7640))
```

Comment:
Maybe change to 
`Updated “Select a Token to Send” screen to always list test network token at the end`